packages/sveltekit/src/client/load.ts

```diff
@@ -0,0 +1,39 @@
+import { captureException } from '@sentry/svelte';
+import { addExceptionMechanism, objectify } from '@sentry/utils';
+import type { ServerLoad } from '@sveltejs/kit';
+
+/**
+ * Wrap load function with Sentry
+ *
+ * @param origLoad SvelteKit user defined load function
+ */
+export function wrapLoadWithSentry(origLoad: ServerLoad): ServerLoad {
+  return new Proxy(origLoad, {
+    apply: async (wrappingTarget, thisArg, args: Parameters<ServerLoad>) => {
+      try {
+        return await wrappingTarget.apply(thisArg, args);
```

Comment:
also here, let's return the original promise and handle the error catching separately?